apps/src/jigsaw/levels.js

```diff
@@ -23,12 +23,16 @@ var jigsawBlockWithDeletableAttr = function (
   if (childType === 'statement') {
     childAttr = " name='child'";
   }
+  // We add a static block ID to the first block in the lesson
+  // so that we can validate whether or not it is clicked.
+  const idAttr = type === 'jigsaw_1A' ? ' id="block1"' : '';
```

Comment:
Oh, or maybe it would make sense to pull this "block1" into a constants file with an explanation and reference it both here and above? Just a thought.